src/md2hast.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { unified } from "unified";
+
+import { md2hast, preprocessor } from "./index.js";
+
+const parse = (raw: string) => {
+  const parser = unified().use(md2hast);
+
+  return parser.runSync(parser.parse(preprocessor(raw)));
+};
+
+describe("Markdown", () => {
+  it("supports heading", () => {
```

Comment:
```suggestion
  it("supports basic markdown", () => {
```

I think it reflects better what the actual test does.